sdk/agrifood/azure-verticals-agrifood-farming/README.md

```diff
@@ -32,27 +32,21 @@ Use FarmBeats client library for Python to do the following.
 
 ### Authenticate the client
 
-#### Using Azure Active Directory
+#### Using Microsoft Entra ID
 
-In order to interact with the Azure FarmBeats service, your client must present an Azure Active Directory bearer token to the service.
+In order to interact with the Azure FarmBeats service, your client must present a Microsoft Entra ID bearer token to the service.
 
-The simplest way of providing a bearer token is to use the `DefaultAzureCredential` authentication method by providing client secret credentials is being used in this getting started section but you can find more ways to authenticate with [azure-identity][azure_identity].
+The best way of providing a bearer token is to use the `DefaultAzureCredential` with a [managed identity](https://learn.microsoft.com/azure/developer/java/sdk/authentication/azure-hosted-apps#authenticate-a-user-assigned-managed-identity-with-defaultazurecredential). For development scenarios, `DefaultAzureCredential` uses a logged-in developer tool's credentials. For more information about developer credentials, see [here](https://learn.microsoft.com/azure/developer/java/sdk/authentication/dev-env). For other authentication flows, see [here](https://learn.microsoft.com/azure/developer/java/sdk/authentication/overview).
```

Comment:
Let's recommend that the customer use `ManagedIdentityCredential` instead of DAC. That's the direction we'd like to start pushing customers.